src/donationswap.py

```diff
@@ -773,6 +773,7 @@ def approve_match(self, secret):
 		match, old_offer, new_offer, my_offer, _ = self._get_match_and_offers(secret)
 
 		if match is None:
+			# this error is shown directly to the user, don't put any sensitive details in it!
```

Comment:
_all_ things that come out of `util.Template` are shown to the user ;-)

but yeah, I agree: better safe than sorry.

Reply:
fair, will try to remember that, i'll leave them since it isn't really an issue :)